hamilton/plugins/polars_extensions.py

```diff
@@ -200,7 +202,7 @@ class PolarsCSVWriter(DataSaver):
 
     @classmethod
     def applicable_types(cls) -> Collection[Type]:
-        return [DATAFRAME_TYPE]
+        return [DATAFRAME_TYPE, pl.LazyFrame]
```

Comment:
Can then add this for all the other polars loaders. A few were just merged, so you might want to rebase from main if you haven't already.